packages/datadog-plugin-jest/test/circus.spec.js

```diff
@@ -173,7 +173,13 @@ describe('Plugin', function () {
             expect(testSpan.service).to.equal('test')
             expect(testSpan.resource).to.equal(`packages/datadog-plugin-jest/test/jest-test.js.${name}`)
             expect(testSpan.meta[TEST_FRAMEWORK_VERSION]).not.to.be.undefined
-          }, { timeoutMs: assertionTimeout })
+          }, {
+            timeoutMs: assertionTimeout,
+            traceMatch: (traces) => {
```

Comment:
Do we need something as flexible? Just passing a span name would probably be enough, since unrelated spans would have other names.

Reply:
a function and a string is not going to be a big difference in code, but right now we don't need the function so I'll remove it. If we ever need it, it's an easy change

Reply:
yeah I started with string match but it was not great for some cases, and a regex seemed like a good alternative